doc/modules/cassandra/pages/reference/cql-commands/create-table-examples.adoc

```diff
@@ -68,6 +68,28 @@ After the disk space limit is reached, writes to CDC-enabled tables are rejected
 See https://docs.datastax.com/en/dse/6.8/dse-admin/datastax_enterprise/config/configCassandra_yaml.html#configCassandra_yaml__cdcSpaceSection[Change-data-capture (CDC) space settings] for information about available CDC settings.
 ====
 
+== Create a table with a trie memtable
+
+To create a table with a trie memtable, a memtable configuration must be enabled in `cassandra.yaml`. See xref:architecture/storage-engine/memtable.adoc[Memtable] for more information.
+
+To create a table with a trie memtable:
+
+[source,language-cql]
+----
+include::cassandra:example$CQL/cyclist_id-table.cql[tag=trie-memtable]
+----
+
+== Create a table with a sharded skiplist memtable
+
+To create a table with a sharded skiplist memtable, a memtable configuration must be enabled in `cassandra.yaml`. See xref:architecture/storage-engine/memtable.adoc[Memtable] for more information.
+
+To create a table with a trie memtable:
```

Comment:
Shouldn't this be "sharded skiplist memtable"?

Reply:
I'm not sure what you are commenting on? Doesn't it say "sharded skiplist memtable" in line 84?

Reply:
Never mind - I realized what the error was. I changed this to a skiplist example anyway.